.check_with_codespell.sh

```diff
@@ -0,0 +1 @@
+codespell --skip='.git,version-specific,scripts' --ignore-words-list=atleast,ninjs,simpy,proovread --quiet-level=2; retVal=$?; if [ $retVal -eq 1 ]; then echo 'Got one expected warning, so success!';  else $(exit $retVal); fi;
```

Comment:
I  couldn't find a way to execute this line in a github action so I had to hide it in a script